### PR TITLE
Add DynamoDB tables for Author

### PR DESCRIPTION
**Pull request by samiwel**
This PR adds two new tables for Author. One for the list of questionnaires and another for the version information for each questionnaire.

### CODE REVIEW

**Comment by jonnyshaw89** [comment]
This PR would prevent us from running either Runner or Author independently as the creation of their tables is linked

File: author_questionnaire_versions.tf [hunk 1]
```diff
@@ -0,0 +1,61 @@
+resource "aws_dynamodb_table" "author_questionnaire_versions_table" {
+  name           = "${var.env}-questionnaire-versions"
+  hash_key       = "id"
+  billing_mode   = "PAY_PER_REQUEST"
```
Comment (tgandrews) [inline author_questionnaire_versions.tf:4]: ~~Is this on demand?~~
On demand isn't an option - weirdly.
https://www.terraform.io/docs/providers/aws/r/dynamodb_table.html#billing_mode

**Comment by samiwel** [comment]
@jonnyshaw89 True but aren't the two products intrinsically linked now that they share the same DynamoDB service in AWS? The author terraform repo provisions survey runner as well, so we still need to survey runner tables to get created as well as the author tables?

Is the concern more around deploying runner independently of author? In which case we would almost need to fork this repo into an `eq-author-terraform-dynamodb`?

**Comment by jonnyshaw89** [comment]
yeh it makes sense from an author point of view as author needs runner. but runner does not need author

**Comment by samiwel** [comment]
@jonnyshaw89 @tgandrews I will close this PR, fork the repo and open a PR on the new repo. I will also make a change to eq-author-terraform to point at the new repo.

**Pull request closed without merging**